myfirstreactapp/src/Organisation.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+export default class Organisation extends Component {
+  render() {
+    const website=this.props.website;
+    // const [email\r]=this.props.["email\r"];
+    const tel=this.props.tel;
+    const area=this.props.area;
+    const process=this.props.process;
+    const name=this.props.name;
+    const clients=this.props.clients;
+    const services=this.props.services;
+    const borough=this.props.borough;
+    const type=this.props.type;
+    const day=this.props.day;
+    return (
+      <div className='test'>
```

Comment:
`test` isn't an ideal class name for an html element here. It does nothing to explain what this element is!